.circleci/config.yml

```diff
@@ -4,10 +4,13 @@ jobs:
   build:
     working_directory: ~/openreview-matcher-repo
     docker:
-    - image: circleci/node:8.9.1-stretch
-    - image: circleci/redis:3-stretch
-    - image: circleci/mongo:3.6-stretch
-    - image: elastic/elasticsearch:6.2.2
+    - image: circleci/node:12.13.1
```

Comment:
It won't affect anything but I think we should start upgrading to node 12.18.3 which is the LTS version. I'll start making this change in other repos as well :D

Reply:
Ok, done

Reply:
@carlosmondra Looks like CircleCi is not able to fetch an fetch an image for node 12.18.3 and failed.

Reply:
Oh interesting, then leave it as it was before. I'll merge it after that. Thanks for checking though :)